glsk/glsk-document-cim/src/test/java/com/powsybl/glsk/api/util/converters/GlskPointScalableConverterTest.java

```diff
@@ -65,6 +66,12 @@ public void testConvertGlskPointToScalableB45MeritOrder() {
         Scalable scalable = GlskPointScalableConverter.convert(testNetwork, glskMeritOrder);
         double done = scalable.scale(testNetwork, 100.0);
         Assert.assertTrue(DoubleMath.fuzzyEquals(6, done, 0.0001));
+        Generator generator1 = testNetwork.getGenerator("FFR1AA1 _generator");
+        Assert.assertTrue(DoubleMath.fuzzyEquals(2001., generator1.getTargetP(), 0.0001));
```

Comment:
The fuzzyEquals is equivalent to the assertEquals ( Math.abs(a - b) <= tolerance || Double.valueOf(a).equals(Double.valueOf(b)). I used it because it was already used in the same method.

Reply:
The fuzzyEquals is equivalent to the assertEquals ( Math.abs(a - b) <= tolerance || Double.valueOf(a).equals(Double.valueOf(b)). I used it because it was already used in the same method.